ext/sockets/tests/mcast_ipv6_recv_limited.phpt

```diff
@@ -8,7 +8,12 @@ sockets
 if (!defined('IPPROTO_IPV6')) {
     die('skip IPv6 not available.');
 }
-$s = socket_create(AF_INET6, SOCK_DGRAM, SOL_UDP);
+// hide the output from socket_create() because it can raise
+// a warning if (for example) the linux kernel is lacking ipv6
+@$s = socket_create(AF_INET6, SOCK_DGRAM, SOL_UDP);
+if ($s === false) {
+  die("skip unable to create socket");
```

Comment:
```suggestion
    die("skip unable to create socket");
```
Nit. Yes, I see the existing code above uses 2 but we still use 4 everywhere else :slightly_smiling_face:

Reply:
No problem, fixed, thanks!